plaid/institutions.go

```diff
@@ -7,14 +7,15 @@ import (
 )
 
 type Institution struct {
-	Credentials  []Credential `json:"credentials"`
-	HasMFA       bool         `json:"has_mfa"`
-	ID           string       `json:"institution_id"`
-	MFA          []string     `json:"mfa"`
-	Name         string       `json:"name"`
-	Products     []string     `json:"products"`
-	CountryCodes []string     `json:"country_codes"`
-	OAuth        bool         `json:"oauth"`
+	Credentials    []Credential `json:"credentials"`
+	HasMFA         bool         `json:"has_mfa"`
+	ID             string       `json:"institution_id"`
+	MFA            []string     `json:"mfa"`
+	Name           string       `json:"name"`
+	Products       []string     `json:"products"`
+	CountryCodes   []string     `json:"country_codes"`
+	OAuth          bool         `json:"oauth"`
+	RoutingNumbers []string     `json:"routing_numbers,omitempty"`
```

Comment:
I don't think `omitempty` is actually necessary here, since Institutions are only deserialized and `omitempty` would only impact the encoding of the field. Unless I'm missing something?

https://golang.org/pkg/encoding/json/#Marshal

```suggestion
	RoutingNumbers []string     `json:"routing_numbers"`
```

Reply:
Nope, that totally makes sense, thanks for the suggestion! Committed